### PR TITLE
Adding Arabic

### PR DESCRIPTION
**Pull request by m7mdtiger**

### CODE REVIEW

File: frontend/static/languages/_list.json
Comment (rizwanmustafa) [inline frontend/static/languages/_list.json:1]: Please revert the changes on this file. It is going to make the reviewing difficult.

**Review by rizwanmustafa — CHANGES_REQUESTED** [review]

File: frontend/static/languages/arabic.json
Comment (rizwanmustafa) [inline frontend/static/languages/arabic.json:1]: Revert the changes on this file as well.

File: frontend/static/languages/_list.json
Comment (m7mdtiger) [reply frontend/static/languages/_list.json:1]: Thank you for your comment. I've just added "arabic_1k".
The remaining part involves adjusting the formatting of the document by adding a trailing comma at the end and not to the beginning like: 
[ 
  "english"
  ,"english_1k"
  ,"english_5k"
]

Is it still better to revert the changes !?

File: frontend/static/languages/arabic.json
Comment (m7mdtiger) [reply frontend/static/languages/arabic.json:1]: The previous file contains vowel marks and we don't write them on typing!

File: frontend/static/languages/_list.json
Comment (rizwanmustafa) [reply frontend/static/languages/_list.json:1]: Yes

File: frontend/static/languages/arabic.json
Comment (rizwanmustafa) [reply frontend/static/languages/arabic.json:1]: Please create a separate PR for that. An ideal PR addresses a single issue.

Comment (m7mdtiger) [reply frontend/static/languages/arabic.json:1]: Alright, thank you.

File: frontend/static/languages/_list.json
Comment (m7mdtiger) [reply frontend/static/languages/_list.json:1]: Thanks. I've reverted the changes.

**Comment by Miodec** [comment]
Does this list include diacritics? (which then later can be removed by enabling lazy mode)

**Comment by m7mdtiger** [comment]
> Does this list include diacritics? (which then later can be removed by enabling lazy mode)

Yes, that should so it for the arabic.json list, which I have already reverted. What remains in this PR is the addition of the arabic_1k.json list, which contains no diacritics.

**Comment by Miodec** [comment]
All lists should contain diacritics so that the users have the option to either keep them or remove them by enabling lazy mode. Please modify the list to include diacritics.

**Comment by Miodec** [comment]
Still, thats what lazy mode is for. Some people may want to type them. Please add them.

**Comment by m7mdtiger** [comment]
> Still, thats what lazy mode is for. Some people may want to type them. Please add them.

Than you. But then the default mode for Arabic should be the lazy mode for all users. Additionally, it's important to consider that the usage of diacritics varies based on the grammatical context, which would require additional time to prepare the list. It may be necessary to allocate a certain percentage for each context, but this task might need more time that I don't have right now!

**Pull request closed without merging**